backend/learn/tests/test_credits.py

```diff
@@ -0,0 +1,104 @@
+from django.test import TestCase
+from learn.models import Block, Task, Level, Student, Toolbox, TaskSession
+from learn.credits import get_credits, get_earned_credits, \
+    total_credits_for_levels
+
+import pytest
+
+
+class BlockTestCase(TestCase):
+    def test_blocks_exists(self):
+        assert Block.objects.exists()
+
+    def test_blocks_are_ordered(self):
+        first_retrieved_blocks = list(Block.objects.all())[:3]
+        first_expected_blocks = [
+            Block.objects.get(name='fly'),
+            Block.objects.get(name='shoot'),
+            Block.objects.get(name='repeat')]
+        assert first_retrieved_blocks == first_expected_blocks
+
+    def test_str_returns_name(self):
+        carrot_block = Block(name='carrot', order=4)
+        assert str(carrot_block) == 'carrot'
+
+
+@pytest.fixture
+def toolbox():
+    return Toolbox(name="some toolbox")
+
+
+@pytest.fixture
+def level(toolbox):
+    return Level(level=5, name="some level", toolbox=toolbox, credits=20)
+
+
+@pytest.fixture
+def task(level):
+    return Task(name="some task", level=level, setting="", solution="")
+
+
+@pytest.fixture
+def student():
+    return Student(user=None)
+
+
+@pytest.mark.django_db
+@pytest.fixture(scope='function')
+def solved_task_for_student():
+    toolbox = Toolbox(name="some toolbox")
+    toolbox.save()
```

Comment:
Can be written as a single command:
`toolbox = Toolbox.objects.create(name="some toolbox")`
(same for other entities below).